src/tx-format/policy.md

```diff
@@ -44,11 +44,12 @@ Transaction is invalid if:
 
 ## `MaxFee`
 
-| name      | type     | description                                                     |
-|-----------|----------|-----------------------------------------------------------------|
-| `max_fee` | `uint64` | The maximum fee payable by this transaction using `BASE_ASSET`. |
+| name      | type     | description                                                                                                                                |
+|-----------|----------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| `max_fee` | `uint64` | The maximum fee payable by this transaction using `BASE_ASSET`. This is used to check transactions before the actual `gas_price` is known. |
 
 Transaction is invalid if:
 
+- `max_fee` is not set
```

Comment:
Maybe it is better to move this check to where the gas price was.

Reply:
I'm not sure I understand. Move this section to where the `## GasPrice` section was? Which "gas price" are you referring to?